src/main/java/cinema/util/HibernateUtil.java

```diff
@@ -0,0 +1,24 @@
+package cinema.util;
+
+import cinema.exception.DataProcessingException;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new DataProcessingException("Couldn't create session factory", e);
```

Comment:
in this case we should use different type of exception compare to Dao layer, let's use just Runtime here